app/workers/send_email_worker.rb

```diff
@@ -4,13 +4,21 @@ class SendEmailWorker < ApplicationWorker
   RATE_LIMIT_THRESHOLD = 21_600 # max requests in a minute, equates to 350 a second
   RATE_LIMIT_INTERVAL = 60
 
-  sidekiq_options retry: 9
-
   sidekiq_retries_exhausted do |msg|
     Email.find_by(id: msg["args"].first, status: :pending)
          &.update!(status: :failed)
   end
 
+  sidekiq_options lock: :until_executed,
```

Comment:
```suggestion
  sidekiq_options lock: :until_executing,
```

This was explained in https://github.com/alphagov/email-alert-api/commit/7d7b1b9ff43e0eb8567b2f63c90684c03433b506.

Reply:
Ah I thought it was a typo in the card as `executed` made more sense to me, didn't realise it came with it's own issues. Corrected.